openpype/hosts/traypublisher/plugins/publish/collect_explicit_colorspace.py

```diff
@@ -46,12 +50,31 @@ def apply_settings(cls, project_settings):
         )
 
         if config_data:
+
             filepath = config_data["path"]
             config_items = colorspace.get_ocio_config_colorspaces(filepath)
+            aliases = set()
+            for _, value_ in config_items.items():
+                if value_.get("type") != "colorspace":
+                    continue
+                if not value_.get("aliases"):
+                    continue
+                for alias in value_.get("aliases"):
+                    aliases.add(alias)
+
+            colorspaces = {
+                name for name, data_ in config_items.items()
+                if data_.get("type") == "colorspace"
+            }
```

Comment:
Out of curiosity - why are you suffixing with `_` underscores after `value_` and `data_`. Feels a bit redundant?

Reply:
issue is the Flake8 is flagging it as not recommended name for variable. Underscore is at right so it should not confuse you with non-public and this is more  matter of preference. For example if one letter variable is awful  to use because of search and replace could pick anything, so I am also adding underscore for those `f_` .

Reply:
All these code styling tools just forcing us into evil 👿 - anyway, thanks for the explanation. I just love code as brief and simple as possible and all these `_` are just not my taste I suppose. Anyway, I'm fine with it if nobody else cares.